src/index.html

```diff
@@ -6,9 +6,31 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo">
+        <img src="./images/logo.png" alt="logo">
+      </a>
+      <nav class="nav">
+        <ul class="conteiner">
+          <li><a href="#" class="is-active link_start">Apple</a></li>
```

Comment:
1. use nav as block and inner children as this block elements (according to class names)
2. add classes to `li`
3. links should have the same class selector
4. apply code styling [recommendations](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md) all over the page

```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a href="#" class="is-active nav__link">
              Apple
            </a>
          </li>
```